### DISCUSSION TITLE
Ensure future content releases include all playable content (ex: Ogri'la faction w/ Shartuul's Transporter)

### DISCUSSION THREAD

**Discussion by andovcr** [chromiecraft/chromiecraft#6509] [Suggestions]
I both understand and appreciate that this is a labor of love from a group of volunteers - I am not here to bash the devs.  I just humbly request that future content releases include all content associated with those phases.

On Nov 29th, the Ogri'la faction was released.  There were some hiccups, and of course every release will have bugs to be fixed afterwards - this is not about that.  The Shartuul's Transporter event is completely inaccessible, and to my laymen's eyes has not been worked on yet.  I understand that Ogri'la & Skyguard have some overlap, so other quests/events are being saved for a future release, but to my knowledge, this Shartuul's Transporter event has no interaction with the Skyguard faction.

I know that this event is a small part of the Ogri'la faction, but it is a part of the stated content release, much like a boss in a raid release.  Again, I know there will be bugs, but I don't think CC has released a raid where a boss is completely non-interactable even a month later.

Finally, if the content absolutely cannot be included in the bracket release, then could the staff please offer up an ETA, rather than remaining silent?

**Comment by Nyeriah** [comment]
Hello,

We try to fix all the issues we are able to fix before content is released, however such is not always possible to a number of reasons (issues needing more research/unvailable data at the time, issues not reported, time constraints, system dependencies, etc.)

It is not feasible for us to wait for every single issue to be addressed before content is released unfortunately, for the reasons mentioned above. We address every issue we can with the resources we have available.

Furthermore, we are not silent, it is just of no use to us to make any promise over issue X or Y being fixed or giving any ETA. It will eventually be addressed as every other issue.

Thanks for your time and support.

Regards

**Reply by andovcr** [reply]
Understood Nyeriah, and I get it - I know you and the rest of the team are working hard in your free time to fix bugs.  And you are appreciated - this is not me being a hater.

I guess my request boils down to: if the devs feel confident to give a release date of content, then that content should be accessible, if not bug-free.  I know you can't promise that this event will be available by the next restart, but I do think it is fair to expect that if the content release is focused around a dailies faction, and that release date is communicated a month in advance, then all of the dailies be usable in some manner.  That would be the same as releasing a raid where all of the bosses are able to be fought and receive loot, right?  I suspect the reason to remove Skyguard reputation from the current live dailies, was so that players would have a reason to play the new Skyguard content when it releases, and help report all of the bugs, rather than already be Exalted and ignore it.  To me, that same motivation is present here - I would like the Shartuul's Transporter to be accessible while the Ogri'la dailies are current content.
